docs/com1DFAAlgorithm.rst

```diff
@@ -81,12 +82,55 @@ The mass per particle determination method can be chosen between:
            particles per cell. Finally, using the MPPKR method, the number of particles per cell is independent from
            both cell size and release thickness (``nPPK`` particles per kernel radius is the target value).
 
-The number of particles placed in each release cell is computed according to the ``massPerPart`` or ``nPPK`` depending
+The number of particles placed in each release cell ``nPartPerCell`` is computed according to the ``massPerPart`` or ``nPPK`` depending
 on the ``massPerParticleDeterminationMethod`` chosen and the area and/or release thickness of the cell.
 The number should be an integer meaning that the float is rounded up or down with a probability corresponding to the
 decimal part (i.e. 5.7 will be rounded to 6 with a probability of 0.7 and 5 with a probability of 0.3).
-This ensures a better match with the desired ``massPerPart`` value. Particles are then placed randomly within the
-mesh cell.
+This ensures a better match with the desired ``massPerPart`` value.
+
+There are then different ways to place the particles in the cells. This is decided by the
+``initPartDistType`` parameter in the configuration file:
+
+    - ``random`` initialization:
+      The ``nPartPerCell`` particles are placed randomly within each release cell.
+      This initialization method allows steps of ones in the choice of ``nPartPerCell`` which enables to
+      avoid large jumps in the mass of the particles or the number of particles per Cell between one release cell to another.
+      Due to the random initialization process, some particles cluster can appear. For operational applications
+      this does not seem to have a significant impact since the particles will redistribute in the first
+      few time steps due to the pressure gradient. For some specific research application (eg. the dam break test),
+      the presence of clusters might disturb the results. In this case applying an initialization reprocessing
+      can help (see  ``iniStep`` in the configuration file).
+      The random number generator is controlled by a random seed which ensures the possibility to
+      reproduce the DFA results.
```

Comment:
```suggestion
      The ``nPartPerCell`` particles are placed randomly within each release cell.
      This initialization method allows steps of ones in the choice of ``nPartPerCell``, which enables to
      avoid large jumps in the mass of the particles or the number of particles per Cell between one release cell to another.
      Due to the random initialization process, some particles cluster can appear. For operational applications,
      this does not seem to have a significant impact since the particles will redistribute in the first
      few time steps due to the pressure gradient. For some specific research applications (e.g. the dam break test),
      clusters might disturb the results. In this case, applying an initialization reprocessing
      can help (see  ``iniStep`` in the configuration file).
      The random number generator is controlled by a random seed which ensures the possibility to
      reproduce the DFA results.
```